src/pytest_respect/plugin.py

```diff
@@ -16,7 +16,21 @@ def pytest_addoption(parser):
     group = parser.getgroup("respect")
     group.addoption(
         "--respect-accept",
-        action="store_true",
-        default=False,
+        action="store_const",
+        dest="respect_accept_max",
+        const=1e9,
```

Comment:
There is also math.inf, but then we're introducing floating point which might not be preferable, but it does avoid this very theoretical problem which will probably never happen